server/go/api_apis_service_impl.go

```diff
@@ -35,14 +35,14 @@ func NewApisApiService(kuskClient kusk.Client) ApisApiServicer {
 
 // GetApis - Get a list of APIs
 func (s *ApisApiService) GetApis(ctx context.Context, namespace string, fleetname string, fleetnamespace string) (ImplResponse, error) {
-	apis := &kuskv1.APIList{}
+	var apis *kuskv1.APIList
 	var err error
+
```

Comment:
I declared `apis` here as I need to return it at the end of the method and `err` is because or the calls in lines 42 and 47. I don't like how it looks like either